content/post/know-your-aws-community-day-india-2020-speaker-swaroop-hegde.md

```diff
@@ -0,0 +1,22 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T15:48:24Z
+description = "Swaroop Hegde will be speaking on \"Using Aurora Global Database for 10x Faster API\" at <time> on <date> at AWS Community Day India 2020."
```

Comment:
time/date placeholders

Reply:
Corrected.

Reply:
@dA505819  the description hasn't been updated

Reply:
@SathyaBhat Placeholders already corrected. Checked twice on forestry.io .
Fixed one extra space.